enhancements/tenant-onboarding/README.md

```
@@ -0,0 +1,119 @@
+
+## Requirements
+
+### Functional Requirements
+
+#### Tenant CR Specification
+- **FR-1:** `spec.displayName` as required string field
+- **FR-2:** `spec.emailDomains` as required list of strings field
+- **FR-3:** Kubebuilder validation markers for field requirements
+- **FR-4:** Admission webhook validates email domain format
+
+#### Tenant CR Lifecycle
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)